keps/sig-cli/0031-kustomize-integration.md

```diff
@@ -16,8 +16,8 @@ approvers:
   - "@soltysh"
 editor: TBD
 creation-date: 2018-11-07
-last-updated: yyyy-mm-dd
-status: pending
+last-updated: 2019-01-07
+status: implementable
 see-also:
   - [KEP-0008](https://github.com/kubernetes/community/blob/master/keps/sig-cli/0008-kustomize.md)
```

Comment:
See PR #664 -- this line apparently prevents the table from rendering properly

Reply:
Fixed it. Thank you.